### PR TITLE
Twilio Integration

### PR DESCRIPTION
**Pull request by jiefugong**
This won't be final behavior, but adds sample Twilio integration to send a confirmation text when a user is registered. This will likely be changed later to have delivery instructions, etc.

Please PM me for access to the environment variables which should be stored privately and locally!

This PR also reattaches the user model which we can use to store information about a donation sign up, without forcing the user to actually sign in.

### CODE REVIEW

File: backend/helper_zero/models.py [hunk 1]
```diff
@@ -1,5 +1,16 @@
 from django.db import models
 
+class User(models.Model):
+  name = models.CharField(max_length=120, null=True)
```
Comment (vicky11z) [inline backend/helper_zero/models.py]: get rid of all the null=True if blank is not set

File: backend/helper_zero/serializers.py [hunk 2]
```diff
@@ -1,18 +1,19 @@
 from rest_framework import serializers
-from .models import Organization
-from .models import DonationRequest
+from .models import User, Organization, DonationRequest
+
+class UserSerializer(serializers.ModelSerializer):
```
Comment (vicky11z) [inline backend/helper_zero/serializers.py]: oh also you'll need to specify that zipcode, lat, and lon could be blank. Looks like this:
```
zipcode = serializers.CharField(allow_null=True)
```

Comment (vicky11z) [reply backend/helper_zero/serializers.py]: Can you test creating a user locally with only one of them set?

Comment (jiefugong) [reply backend/helper_zero/serializers.py]: Resolved in person

File: backend/helper_zero/models.py (hunk 1 above)
Comment (jiefugong) [reply backend/helper_zero/models.py]: Resolved in person

**Pull request merged**